interfaces/Config/templates/staticcfg/css/style.css

```diff
@@ -378,6 +378,25 @@ tr.separator {
     padding: 1.5em 2em 3em
 }
 
+/* hacks for multiselect  */
+.overflow {
+    overflow-x: visible !important;
+    overflow-y: visible !important;
+}
+div.btn-group.open input[type='checkbox'] {
+    top: 0px !important;
+    margin-left: -22px !important;
+}
+div.btn-group.open label {
+    padding-left: 30px !important;
+    padding-bottom: 5px !important;
+}
+.multiselect.disabled {
```

Comment:
Why this extra disabled styling? I prefer the default Bootstrap disabled styling much better.
Hacked example:
![image](https://user-images.githubusercontent.com/5703454/234593871-bf2da402-b8be-42be-95ee-17a274ac0277.png)

Reply:
it was to combat sab's style.css where we override bootstrap's .disabled with #aaa which i find makes it a little hard to read (and was to help jcfp's issue). but i can tweak it so its not so stark..  

rather than use disabled, shouldnt we just use readonly instead?